Modules/_elementtree.c

```diff
@@ -3259,6 +3259,11 @@ xmlparser_init(PyObject *self, PyObject *args, PyObject *kwds)
         PyErr_NoMemory();
         return -1;
     }
+    /* expat < 2.1.0 has no XML_SetHashSalt() */
+    if (EXPAT(SetHashSalt) != NULL) {
+        EXPAT(SetHashSalt)(self_xp->parser,
```

Comment:
This is the only line that has been modified from the previous PR's. Namely it's `self_xp->parser `on 3.4. On the other branches it's `self->parser` due to argumentclinication.